articles/key-vault/vs-secure-secret-appsettings.md

```diff
@@ -143,7 +145,7 @@ Follow instructions from ASP.NET core section to configure a Key Vault for your
 
 1. Install the following NuGet package to your project
 ```
-Microsoft.Configuration.ConfigurationBuilders.Azure.1.0.0-alpha1.nupkg
+Microsoft.Configuration.ConfigurationBuilders.Azure.1.0.0-preview.nupkg
```

Comment:
Again, -preview2 is the latest at the moment... and it looks like the latest version of this doc has already been updated for Azure Key Vault.